### PR TITLE
Updating the website information

### PR DESCRIPTION
**Pull request by anaschwendler**
**What was updated:**
- [x] Home: finished
- [x] Articles: added the new text
- [x] About: updated the information
- [x] FAQ: updated

This is the first update when all doubts solved, I'll finish this PR.

**Who can review this PR?**
@cuducos @Irio @jtemporal 

**What is missing?**
- [x] English version of the updated texts

### CODE REVIEW

File: _i18n/en.yml [hunk 1]
```diff
@@ -19,6 +19,7 @@ footer:
   contact: Contact
   partners: Partners
 stories:
+  intro: Our biggest commitment as a project is to make public data accessible to the anyone. Read below our independent productions based on what we work.
```
Comment (cuducos) [inline _i18n/en.yml:23]: As a project, our biggest commitment is to make public data accessible to the anyone. Following are our independent production based on the work we do.

File: _i18n/en.yml [hunk 2]
```diff
@@ -42,47 +43,44 @@ explore:
...
-  paragraph_four: On the 7 of September the first crowdfunding campaign for the project goes live asking for money to pay for the team work full time during 2 months. In the end the campaign raised 20% more than the original target, summing up more than 25,000.00 USD.
-  paragraph_five: 'We officially started the job in early November and since then the team holds daily meetings and has published tons of fully open source content — that is to say, open for the whole community. This makes part of our guideline: it is more than a technology project, our goal it is to spread information and political conscience. We want to settle a new age of open data and its public access.'
+  paragraph_one: We are a technology project that uses artificial intelligence to audit public accounts and support social control. The idea came from the data scientist <a href="https://twitter.com/irio"> Irio Musskopf </a>, as a way to actively participate in the democratic process, overseeing public spending. When sharing the idea with friends, it didn't take long for the project be spread all over the country, with a team of 10 people and more than 600 volunteers willing to do it.
```
Comment (cuducos) [inline _i18n/en.yml:47]: Serenata de Amor is a technological project that uses artificial intelligence for social accountability of public expenses. The idea came from the data scientist <a href="https://twitter.com/irio">Irio Musskopf</a>. In the first place the purpose was to find a way to actively participate in the democratic process overseeing public spending. After he's shared the idea with some friends, the project actually started to happen with a dedicated team of 10 people and more than 600 volunteers from over Brazil.

File: _i18n/en.yml [hunk 3]
```diff
@@ -42,47 +43,44 @@ explore:
...
+  paragraph_two: And then, the Operation Serenade of Love has started, focused on monitoring, with the help of technology, the reimbursements made by the Parliamentary Activity Charge (CEAP) - money that costs food, transportation, lodging and even expenses with culture and TV signatures of parliamentarians.
```
Comment (cuducos) [inline _i18n/en.yml]: This as the very beginning of Operation Serenata de Amor. We started focused on monitoring, with the help of technology, reimbursements made by the Quota for the Exercise of the Parliamentary Activity (CEAP, as in the Portuguese acronym) from Brazilian Federal Chamber of Deputies. This money is allocated in order to fund meals, transportation, lodging and even expenses with culture and paid TV for congresspeople.

File: _i18n/en.yml [hunk 4]
```diff
@@ -42,47 +43,44 @@ explore:
...
+  paragraph_three: On September 7, 2016, our first collective financing campaign was alive, which managed to raise approximately R$ 80,000, 30% more than the amount requested, enough to finance the operations of the team for the first 90 days.
```
Comment (cuducos) [inline _i18n/en.yml]: In 2016, precisely on the September 7th, our first crowdfunding campaign was launched and we raised approximately R$ 80,000 – 30% more than our initial target. That allowed us to afford the dedicated team for the first 90 days.

File: _i18n/en.yml [hunk 5]
```diff
@@ -42,47 +43,44 @@ explore:
...
+  paragraph_four: Since then, our work, based on information and transparency, has gained space in events and lectures around the world, and has been prominent in the media, being covered by more than 90 journalistics vehicles, including the largest in the country, such as <a href="https://globoplay.globo.com/v/6267741/programa/" target="_blank">Fantastico</a>, <a href="https://globoplay.globo.com/v/5608268/" target="_blank">Jornal Nacional</a> and <a href="http://www1.folha.uol.com.br/poder/2017/01/1852180-jovens-criam-robo-que-monitora-despesas-de-deputados-federais.shtml" target="_blank">Folha de S. Paulo</a>, and a great relevance in the scenario of social control in Brazil.
```
Comment (cuducos) [inline _i18n/en.yml]: So far we have been working based on information and transparency. The importance of our work has been acknowledged in numerous invitations to speak in events all around the world and in numerous appearances in the media – including more than 90 different news outlets, counting Brazilian largest ones, such as <a href="https://globoplay.globo.com/v/6267741/programa/" target="_blank">Fantástico</a>, <a href="https://globoplay.globo.com/v/5608268/" target="_blank">Jornal Nacional</a> and <a href="http://www1.folha.uol.com.br/poder/2017/01/1852180-jovens-criam-robo-que-monitora-despesas-de-deputados-federais.shtml" target="_blank">Folha de S. Paulo</a>. In sum we are afraid we have a considerable relevance in social accountability of public expenses in Brazil.

**Review by cuducos — CHANGES_REQUESTED** [review]
Some suggestions added in addition to yesterday comment, but I still have to review the FAQ, ok?

File: _i18n/pt-br.yml [hunk 6]
```diff
@@ -43,47 +45,44 @@ explore:
...
-  question_eighteen: 18. Posso usar o código da Serenata de Amor na minha cidade, no meu estado ou no meu país?
-  answer_eighteen: Claro, essa ideia é sensacional — e é um dos principais motivos de adotarmos código aberto. <a href="https://medium.com/data-science-brigade/expandindo-a-serenata-de-amor-para-outras-esferas-48bed204d36d">Esse artigo explica isso pode ocorrer</a>.
+  question_one: 1. O que o nome Serenata de Amor significa?
+  answer_one: 'Os motivos são três:<br> 1 - Nos inspiramos diretamente no famoso Caso Toblerone - um famoso acontecimento onde uma política na Suécia renunciou o seu cargo após ser flagrada com um simples Toblerone na fatura de seu cartão corporativo. Queremos fazer isso: encontrar corrupção em pequenos gastos, mas em volume grande.<br> 2 - Parece nome de operação da Polícia Federal, e isso é muito legal.<br> 3 - É a nossa Serenata de Amor para o Brasil.'
```
Comment (cuducos) [inline _i18n/pt-br.yml]: > Nos inspiramos diretamente no famoso Caso Toblerone - um famoso acontecimento […]

Is necessary this repetition of _famoso_? When reviewing the English version I removed it. I suggest we do the same in the Portuguese version.

File: _i18n/en.yml [hunk 7]
```diff
@@ -42,47 +43,44 @@ explore:
...
-  question_eighteen: 18. Can I use the source code of Serenata de Amor in my city, state or country?
-  answer_eighteen: Sure, this idea is awesome — and also this idea is one of the main purposes of our adoption of open source.
+  question_one: 1. What does the name Serenata de Amor mean?
+  answer_one: 'There three reasons: <br>1 - We are  directly inspired by the famous Toblerone Affair – a famous case in which a politician in Sweden resigned after being caught with a simple Toblerone in the invoice of her corporate card. We want to do this: find corruption at small scale, but in large volume. <br>2 - It seems to be the a nonsense operation name typical of the Brazilian Federal Police, and this is very cool. <br>3 - It literally means love serenade — so this is our love serenade to Brazil.'
```
Comment (cuducos) [inline _i18n/en.yml]: There three reasons: 

1 - We were directly inspired by the Toblerone Affair – a famous case in which a politician in Sweden resigned after being caught with a simple Toblerone she had paid with the taxpayer money. That is what we want: to monitor public expenses as small as a chocolate bar.

2 - It sounds like a nonsense operation name typical in the Brazilian Federal Police, and this is funny.
 
3 - Literally it means “love serenade” — so this is our love serenade to Brazil.

File: _i18n/pt-br.yml (hunk 6 above)
Comment (cuducos) [reply _i18n/pt-br.yml]: Yet I raise the question: we are not focusing in _corruption_ anymore, so I find this answer quite unfit for our present moment. I suggested something different in the English review.

File: _i18n/en.yml [hunk 8]
```diff
@@ -42,47 +43,44 @@ explore:
...
+  question_two: 2. Who is responsible for the project?
+  answer_two: 'We are a <a href="https://twitter.com/cuducos/status/819520770716090368" target="_blank"> group </a> of technology enthusiasts. The project started as an initiative of the data scientist <a href="twitter.com/irio" target="_blank">Irio Musskopf</a>, grew and today has taken on a life of its own, with a team of 10 professionals. Every code is open, and now we have hundreds of volunteers: collaborators from all over the world, including journalists and researchers.'
```
Comment (cuducos) [inline _i18n/en.yml]: We are a  group of technology enthusiasts. The project started from the data scientist <a href="twitter.com/irio" target="_blank">Irio Musskopf</a>'s initiative. It has grown considerably and today we are a team of 10 professionals. Every code is open, and we have hundreds of volunteers: collaborators from all over the world, including journalists and researchers.

**Review by cuducos — CHANGES_REQUESTED** [review]
Extra review, FAQ questions 1 and 2 I think need changes in en-US and pt-BR.

(still pending review from 3 and on)

File: _i18n/en.yml (hunk 4 above)
Comment (anaschwendler) [reply _i18n/en.yml]: In 2016, precisely on the September 7th, Independence Day in Brazil, our first crowdfunding campaign was launched and we raised approximately R$ 80,000 – 30% more than our initial target. That allowed us to afford the dedicated team for the first 90 days.

File: _i18n/pt-br.yml (hunk 6 above)
Comment (anaschwendler) [reply _i18n/pt-br.yml]: Completely agree, I was changing the questions that had that word to social control, but that one passed by me; Thanks @cuducos

**Comment by anaschwendler** [comment]
I finished the commit by Friday, but forgot to push the commit!

Thanks @Irio for reminding me that!
@cuducos that is the final version that we discussed on the document.

**Review by Irio — APPROVED** [review]

**Pull request merged**